cocos/core/geometry/distance.ts

```diff
@@ -39,26 +39,26 @@ const e = new Array(3);
 
 /**
  * @en
- * the distance between a point and a plane
+ * Calculates the distance between a point and a plane.
  * @zh
  * 计算点和平面之间的距离。
- * @param {Vec3} point @en Target point @zh 目标点。
- * @param {Plane} plane @en Target plane @zh 目标平面。
- * @return @en The distance in between @zh 距离。
+ * @param point @en The target point. @zh 目标点。
+ * @param plane @en The target plane. @zh 目标平面。
+ * @returns @en The distance between the point and the plane. @zh 点和平面之间的距离。
```

Comment:
I just wonder using `a` is better. For example
`The distance between a point and a plane.`